mage_ai/streaming/sources/nats_js.py

```diff
@@ -165,24 +196,36 @@ def batch_read(self, handler: Callable):
             self.stop_loop()
 
     def read(self, handler: Callable):
-        self.init_client()
```

Comment:
why do you not call `init_client`?

Reply:
because it's already called by the init method of BaseSource class